SocialCareCaseViewerApi.Tests/V1/Boundary/Request/QueryCaseSubmissionsRequestTests.cs

```diff
@@ -18,9 +18,9 @@ public class QueryCaseSubmissionsRequestTests
             new object?[] { TestHelpers.CreateQueryCaseSubmissions(submissionStates: new List<string>{"submission-state"}), true, null},
             new object?[] { TestHelpers.CreateQueryCaseSubmissions(createdBefore: DateTime.Now), true, null},
             new object?[] { TestHelpers.CreateQueryCaseSubmissions(createdAfter: DateTime.Now), true, null},
+            new object?[] { TestHelpers.CreateQueryCaseSubmissions(ageContext: "A"), true, null},
```

Comment:
is there a reason for the new filter above the two we added earlier? or just a personal preference?

Reply:
Nah, it's just because I already wrote this then merged development into my branch which then had merge conflicts.

^ also the reason for the diff question about the formatting stuff